twake/frontend/src/app/state/recoil/atoms/CurrentCompany.ts

```diff
@@ -0,0 +1,16 @@
+import { atom, AtomEffect } from "recoil";
+
+import { CompanyType } from "app/models/Company";
+import UserContextState from "../../UserContextState";
+
+const currentWorkspaceEffect: AtomEffect<CompanyType| undefined> = ({ onSet }) => {
+  onSet(company => UserContextState.company = company);
+};
+
+export const CurrentCompanyState = atom<CompanyType | undefined>({
+  key: 'CurrentCompanyState',
+  default: undefined,
+  effects_UNSTABLE: [
```

Comment:
After all this is just a question of is it more unstable than current Twake or less unstable :p

Reply:
You get it 😸